scipy/stats/_stats_py.py

```diff
@@ -1877,122 +1699,36 @@ def normaltest(a, axis=0, nan_policy='propagate'):
-    >>> _ = ax.annotate(annotation, (13.5, 5e-4), (14, 5e-3), arrowprops=props)
-    >>> i = stat_vals >= res.statistic  # index more extreme statistic values
-    >>> ax.fill_between(stat_vals[i], y1=0, y2=pdf[i])
-    >>> ax.set_xlim(8, 16)
-    >>> ax.set_ylim(0, 0.01)
-    >>> plt.show()
-    >>> res.pvalue
-    0.0014779023013100172
-
-    If the p-value is "small" - that is, if there is a low probability of
-    sampling data from a normally distributed population that produces such an
-    extreme value of the statistic - this may be taken as evidence against
-    the null hypothesis in favor of the alternative: the weights were not
-    drawn from a normal distribution. Note that:
-
-    - The inverse is not true; that is, the test is not used to provide
-      evidence for the null hypothesis.
-    - The threshold for values that will be considered "small" is a choice that
-      should be made before the data is analyzed [4]_ with consideration of the
-      risks of both false positives (incorrectly rejecting the null hypothesis)
-      and false negatives (failure to reject a false null hypothesis).
-
-    Note that the chi-squared distribution provides an asymptotic
-    approximation of the null distribution; it is only accurate for samples
-    with many observations. This is the reason we received a warning at the
-    beginning of the example; our sample is quite small. In this case,
-    `scipy.stats.monte_carlo_test` may provide a more accurate, albeit
-    stochastic, approximation of the exact p-value.
-
-    >>> def statistic(x, axis):
-    ...     # Get only the `normaltest` statistic; ignore approximate p-value
-    ...     return stats.normaltest(x, axis=axis).statistic
-    >>> res = stats.monte_carlo_test(x, stats.norm.rvs, statistic,
-    ...                              alternative='greater')
-    >>> fig, ax = plt.subplots(figsize=(8, 5))
-    >>> plot(ax)
-    >>> ax.hist(res.null_distribution, np.linspace(0, 25, 50),
-    ...         density=True)
-    >>> ax.legend(['aymptotic approximation (many observations)',
-    ...            'Monte Carlo approximation (11 observations)'])
-    >>> ax.set_xlim(0, 14)
-    >>> plt.show()
-    >>> res.pvalue
-    0.0082  # may vary
-
-    Furthermore, despite their stochastic nature, p-values computed in this way
-    can be used to exactly control the rate of false rejections of the null
-    hypothesis [5]_.
-
+    >>> rng = np.random.default_rng()
+    >>> pts = 1000
+    >>> a = rng.normal(0, 1, size=pts)
+    >>> b = rng.normal(2, 1, size=pts)
+    >>> x = np.concatenate((a, b))
+    >>> k2, p = stats.normaltest(x)
+    >>> alpha = 1e-3
+    >>> print("p = {:g}".format(p))
+    p = 8.4713e-19
+    >>> if p < alpha: # null hypothesis: x comes from a normal distribution
+    ...     print("The null hypothesis can be rejected")
+    ... else:
+    ...     print("The null hypothesis cannot be rejected")
+    The null hypothesis can be rejected
```

Comment:
Let's change this to just:
```python3
res = stats.normaltest(x)
res.statistic
<blah>
res.pvalue
<blah>
```
(If we were going to interpret the p-value, I'd the significance threshold should be spelled out in text.)